src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -0,0 +1,881 @@
+        >>> tagged_table = TaggedTable({"date": ["01.01", "01.02", "01.03", "01.04"], "col1": ["a", "b", "c", "a"]}, "col1" )
+        >>> timeseries = TimeSeries._from_tagged_table(tagged_table, time_name = "date")
+        """
+        if time_name not in tagged_table.column_names:
+            raise ValueError(f"Could not find Column '{time_name}'.")
+        table = tagged_table._as_table()
+        # make sure that the time_name is not part of the features
+        result = object.__new__(TimeSeries)
+        feature_names = tagged_table.features.column_names
+        if time_name in feature_names:
+            feature_names.remove(time_name)
+
+        if time_name == tagged_table.target.name:
+            raise ValueError(f"Column '{time_name}' cannot be both time column and target.")
+
+        result._data = table._data
+        result._schema = table.schema
+        result._time = table.get_column(time_name)
+        result._features = table.keep_only_columns(feature_names)
+        result._target = table.get_column(tagged_table.target.name)
+        return result
+
+    @staticmethod
+    # idk if this method should exist, because we cant use it like the Method in TaggedTable, but the Method is static and this seem to doesnt get checked by the megalinter.
+    def _from_table_to_time_series(
+        table: Table,
+        target_name: str,
+        time_name: str,
+        feature_names: list[str] | None = None,
+    ) -> TimeSeries:
+        """Create a TimeSeries from a table.
+
+        Parameters
+        ----------
+        table : Table
+            The table.
+        target_name : str
+            Name of the target column.
+        time_name: str
+            Name of the date column.
+        feature_names : list[str] | None
+            Names of the feature columns. If None, all columns except the target column are used.
+        Retruns
+        -------
+        time_series : TimeSeries
+            the created time series
+
+        Raises
+        ------
+        UnknownColumnNameError
+            If target_name matches none of the column names.
+        Value Error
+            If no feature columns are specified
+
+        Examples
+        --------
+        >>> from safeds.data.tabular.containers import Table, TimeSeries
+        >>> table = Table({"date": ["01.01", "01.02", "01.03", "01.04"], "f1": ["a", "b", "c", "a"], "t": [1,2,3,4]})
+        >>> timeseries = TimeSeries._from_table_to_time_series(table, "t", "date", ["f1"])
+        """
+        if feature_names is None:
+            feature_names = table.column_names
+            if time_name in feature_names:
+                feature_names.remove(time_name)
+            if target_name in feature_names:
+                feature_names.remove(target_name)
+        tagged_table = TaggedTable._from_table(table=table, target_name=target_name, feature_names=feature_names)
+        # check if time column got added as feature column
```

Comment:
I would expect a ValueError just like if a column is specified as both target and feature column

Reply:
it raises one in the constructor of the parent class

Reply:
The `TimeSeries._from_tagged_table` just removes the column from the features if it is declared as a time column. While this is expected behavior (at least I think it is) in the _from_tagged_table method, we specifically define the time and feature columns in this method, so I would expect a ValueError if we defined one column as both time and feature column.

Reply:
yeah but this does not happen here
I create a list with all names and remove it after, the same happens in the ```tagged table``` class in the ```from_table``` function, because when the user does not specifie any features every column is taken as a feature so we gotta remove the target and time column